package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "ampersand-calendar",
+  "version": "0.1.0",
+  "description": "Creates an SVG calendar using D3.",
+  "main": "ampersand-calendar.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:helpdotcom/ampersand-calendar"
```

Comment:
The url is supposed to be a public URL - I don't know if an ssh url counts as that.